roles/certificates/README.rst

```diff
@@ -0,0 +1,34 @@
+An ansible role to install and update CA certificates.
+
+**Role Variables**
+
+.. zuul:rolevar:: certificates_ca
+   :default: []
+   
+   This is a list wich contains the name and the certificate.
+
+   Example:
+
+   .. code-block:: yaml
+
+      certificates_ca:
+        - name: sample.crt
+          certificate: |
+            -----BEGIN CERTIFICATE-----
+            [...]
+            -----END CERTIFICATE-----
+
+.. zuul:rolevar:: certificates_ca_path
+   :default: /usr/local/share/ca-certificates
+
+   The path where the certificates will be stored.
+
+.. zuul:rolevar:: certificates_ca_package_name
+   :default: ca-certificates
+
+   The Debian package what is needed to install for the certificates.
```

Comment:
Use "that" or "which" instead of "what".

Reply:
Done